.changelog/4427.cfg.md

```diff
@@ -0,0 +1,14 @@
+go: bump tendermint to 0.35
+
+Configuration changes required due to Tendermint changes:
+
+Flags `consensus.tendermint.p2p.max_num_{inbound,outbound}_peers` have
+been replaced with a single `consensus.tendermint.p2p.max_connections`.
+Flag `consensus.tendermint.state_sync.consensus_node` has been removed,
+as it is no longer required.
+
+Previously, when running a sentry node, the node behind the sentry had to
+use `consensus.tendermint.p2p.disable_peer_exchange`, but now it must not
+use that flag.  You must instead set `consensus.tendermint.p2p.max_peers`
+and `consensus.tendermint.p2p.max_connections` to 1, and add the sentry
+node's Tendermint ID to `consensus.tendermint.p2p.whitelisted_peers`.
```

Comment:
I don't know.  Someone from devops should take a look at this, I have no idea how sentries are used in production.  I imagine that the above is only true for the very simplest of setups and we'd still need to export all these configuration options to make more complex setups work.

Reply:
Hm, could add something like `tendermint.p2p.sentry_peers` (or something alike) flag, and it would automatically set the whitelisted peers and set max_connections/max_peers to the number of sentry peers.